client/src/pages/Event.js

```diff
@@ -1,82 +1,95 @@
 
-        } catch(error) {
-            console.log(error);
-        }
+  async function fetchEvent() {
+    try {
+      const res = await fetch(`/api/events/${props.match.params.id}`, {
+        headers: {
+          'x-customrequired-header': headerToSend,
+        },
+      });
+      const resJson = await res.json();
+
+      setEvent(resJson);
+      setIsCheckInReady(resJson.checkInReady);
+    } catch (error) {
+      console.log(error);
     }
+  }
 
-    async function setCheckInReady(e) {
-        e.preventDefault();
-        
-        try {
-            await fetch(`/api/events/${props.match.params.id}`, {
-                method: 'PATCH',
-                headers: {
-                    "Content-Type": "application/json",
-                    "x-customrequired-header": headerToSend
-                },
-            })
-                .then(response => {
-                    if (response.ok) {
-                        setEvent(event);
-                        setIsCheckInReady(!isCheckInReady);
-                    }
-                });
+  async function setCheckInReady(e) {
+    e.preventDefault();
 
-        } catch(error) {
-            setIsLoading(!isLoading);
+    try {
+      await fetch(`/api/events/${props.match.params.id}`, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+          'x-customrequired-header': headerToSend,
+        },
+      }).then((response) => {
+        if (response.ok) {
+          setEvent(event);
+          setIsCheckInReady(!isCheckInReady);
         }
+      });
+    } catch (error) {
+      setIsLoading(!isLoading);
     }
+  }
 
-    useEffect(() => {
-        fetchEvent();
-    }, [isLoading, isCheckInReady]);
+  useEffect(() => {
+    fetchEvent();
+  }, [isLoading, isCheckInReady]);
 
-    return (
-        <div className="flex-container">
-            <div className="event-container">
-                {event && event.location ? (
-                    <div className="event-headers">
-                        <h4>{event.name}</h4>
-                        <p>{moment(event.date).format('dddd, MMMM D, YYYY @ h:mm a')}</p>
-                        <p>{event.location.city}</p>
-                        <p>{event.location.state}</p>
-                    </div>
-                    ) : (
-                        <div>Loading...</div>
-                    )
-                }
+  return (
+    <div className="flex-container">
+      <div className="event-container">
+        {event && event.location ? (
+          <div className="event-headers">
+            <h4>{event.name}</h4>
+            <p>{moment(event.date).format('dddd, MMMM D, YYYY @ h:mm a')}</p>
+            <p>{event.location.city}</p>
+            <p>{event.location.state}</p>
+            {event.project ? (
+              <Link
+                to={`/project/${event.project._id}`}
+                className="create-form-button"
+                style={{
+                  width: '140px',
```

Comment:
<img width="452" alt="Screen Shot 2022-06-27 at 5 10 42 PM" src="https://user-images.githubusercontent.com/53061723/176060149-c1f292fb-1cad-4fe5-bacf-087f104fa1a5.png">
not sure why edit event is being cut off here. 

but it looks like we need to add

`height: fit-content;`

in the css

the rest of this looks great

Reply:
Pushed a commit that might fix it, but I'm not sure. Which browser are you seeing this on? I don't see it on firefox or chrome/edge